.github/workflows/build-and-test-windows.yml

```diff
@@ -96,3 +96,43 @@ jobs:
             echo "One or more matrix jobs failed."
             false
           fi
+  windows-supervisor-service-test:
```

Comment:
Since we're building the Collector binary, could we include this in the workflow in `e2e-tests-windows.yaml` instead? We could also take advantage of the cached binary that way.